packages/upstream-protobuf/bin/protoc-gen-dumpcodegenreq.mjs

```diff
@@ -21,8 +21,20 @@ import { stdin, stdout } from "node:process";
 stdin.pipe(createWriteStream("dumpcodegenreq.binpb"));
 
 // Write a minimal a google.protobuf.CodeGenerateResponse to stdout.
-// This is just the field supported_features set to a bitwise "or" of
-// FEATURE_PROTO3_OPTIONAL and FEATURE_SUPPORTS_EDITIONS. See the code snippet
-// below.
-const minimalResponse = new Uint8Array([16, 3]);
+/*
+import {CodeGeneratorResponse_Feature, CodeGeneratorResponse, Edition} from "@bufbuild/protobuf";
+const res = new CodeGeneratorResponse({
+  supportedFeatures: BigInt(
+    CodeGeneratorResponse_Feature.SUPPORTS_EDITIONS | CodeGeneratorResponse_Feature.PROTO3_OPTIONAL
+  ),
+  minimumEdition: Edition.EDITION_LEGACY,
+  maximumEdition: Edition.EDITION_MAX,
```

Comment:
v27.0 requires these fields in the response now to generate code for Edition 2023. We're only using this in tests.